DQM/HLTEvF/python/HLTTrackingMonitoring_cff.py

```diff
@@ -2,9 +2,9 @@
 
 # tracking monitor
 from DQMOffline.Trigger.TrackingMonitoring_cff import *
-iterHLTTracksMonitoringHLT.doProfilesVsLS   = cms.bool(True)
-iterHLTTracksMonitoringHLT.beamSpot = cms.InputTag("hltOnlineBeamSpot")
-pixelTracksMonitoringHLT.beamSpot = cms.InputTag("hltOnlineBeamSpot")
+doProfilesVsLS   = True
+beamSpot = "hltOnlineBeamSpot"
+beamSpot = "hltOnlineBeamSpot"
```

Comment:
```suggestion
iterHLTTracksMonitoringHLT.doProfilesVsLS = True
iterHLTTracksMonitoringHLT.beamSpot = "hltOnlineBeamSpot"
pixelTracksMonitoringHLT.beamSpot = "hltOnlineBeamSpot"
```
It should be like this, I think.

Reply:
Thank you. It was not showing any error though, strange! I have committed your suggestions, hope this is ok to do.